codetransformer/transformers/exc_patterns.py

```diff
@@ -0,0 +1,51 @@
+from sys import exc_info
+
+from dis import dis
+
+from codetransformer.core import (
+    CodeTransformer,
+    Instruction,
+    ops,
+)
+
+
+def match(match_expr, exc_info):
```

Comment:
Hmm, I would like normal style handlers too work also, you could manually construct a function that checks if it is a subclass of exception (not an instance) and then uses match_exception otherwise fall into this check